src/imap/session.rs

```diff
@@ -17,13 +17,15 @@ use crate::tools;
 /// - Chat-Version to check if a message is a chat message
 /// - Autocrypt-Setup-Message to check if a message is an autocrypt setup message,
 ///   not necessarily sent by Delta Chat.
+/// - Chat-Is-Post-Message to skip it in background fetch or when it is too large
```

Comment:
"or when it is too large" -- is this part true? I don't see a comparison with `download_limit` for usual emails